app/ui/reports_query_hash.py

```diff
@@ -105,7 +107,14 @@ def top_table():
 
         """
         )
-        st.dataframe(df, use_container_width=True)
+        df["Query Text"] = df.apply(
+            lambda row: f"{row['Query Id']}: {row['Query Text'][:10000]}"
```

Comment:
Regular python `str`'s would do the correct thing if you did `"asdf"[:1000]` and the length of the string was less than the offset value. Does the dataframe act differently and throw some ArrayIndexOutOfBounds?

Reply:
Row is a dict and the query text field is a str. Should behave just fine.

Reply:
I missed the first half of the f-string in the if-clause. I thought it was just doing:
```
if len(queryText) > 1000:
  queryText[:1000]
else:
  queryText
```